libraries/core_libs/consensus/include/pbft/step/step.hpp

```diff
@@ -0,0 +1,109 @@
+#pragma once
+#include <cassert>
+#include <memory>
+
+#include "common/types.hpp"
+#include "network/network.hpp"
+
+namespace taraxa {
+
+using time_point = std::chrono::system_clock::time_point;
+
+#define kNullBlockHash blk_hash_t(0)
```

Comment:
this is already defined `static const blk_hash_t kNullBlockHash = blk_hash_t(0);`